strawberry/schema/schema.py

```diff
@@ -216,3 +218,12 @@ def introspect(self) -> Dict[str, Any]:
             raise ValueError(f"Invalid Schema. Errors {introspection.errors!r}")
 
         return introspection.data
+
+    def get_scalar(
+        self, scalar: Type
+    ) -> Union[ScalarWrapper, ScalarDefinition, GraphQLScalarType]:
+        if scalar in DEFAULT_SCALAR_REGISTRY:
+            return DEFAULT_SCALAR_REGISTRY[scalar]
+
+        scalar_definition = scalar._scalar_definition
+        return scalar_definition
```

Comment:
How do we ensure that `get_scalar` is only called on objects with `._scalar_definition`?

Reply:
I doesn't have to be. If you process the scalar before calling `super` then you're just dealing with the raw python type.